gradle_project/build.gradle

```diff
@@ -72,9 +46,6 @@ jar {
 
 task compileWithJava8 (type: com.igormaznitsa.jcp.gradle.JcpTask) {
```

Comment:
maybe we can have a super task to create all the `compileWith` subtasks within a for loop.